src/main/java/utils/units/Units.java

```diff
@@ -26,6 +26,10 @@ public static double rpsToRadsPerSec(double rps) {
         return rpmToRadsPerSec(rpsToRpm(rps));
     }
 
+    public static double rotationsToRad(double rotations) {
+        return rotations / (Math.PI * 2);
+    }
```

Comment:
No need to reinvent the wheel -- this already exists, use WPILib's [`Units.rotationsToRadians()`](https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/math/util/Units.html#rotationsToRadians(double)).
Anyway, your calculation looks wrong -- one rotation should be `2π`, not a number which is less than 1.